cypress/integration/LockedSelector.feature

```diff
@@ -0,0 +1,119 @@
+Feature: In the main page, use the LockedSelector to navigate
+
+  # Main page
+
+  Scenario: Notifying that you need to select org unit and program to get started
```

Comment:
In each feature we are having multiple scenarios. It has nothing to do with the tests itself. It can be changed any time and nothing will break.